buildpack.md

```diff
@@ -917,11 +881,11 @@ For each process, the buildpack:
 - MUST specify a `type`, which:
   - MUST NOT be identical to other process types provided by the same buildpack.
   - MUST only contain numbers, letters, and the characters ., _, and -.
-- MUST specify a `command` that is either:
-  - A command sequence that is valid when executed using the shell, if `args` is not specified.
-  - A path to an executable or the file name of an executable in `$PATH`, if `args` is a list with zero or more elements.
-- MAY specify an `args` list to be passed directly to the specified executable.
-- MAY specify a `direct` boolean that bypasses the shell.
+- MUST specify a `command` list such that:
+  - The first element of `command` is a path to an executable or the file name of an executable in `$PATH`.
+  - Any remaining elements of `command` are arguments that are always passed directly to the executable.
+- MAY specify an `args` list to be passed directly to the specified executable, after arguments specified in `command`.
+  - The `args` list is a default list of arguments that may be overridden by the user.
```

Comment:
Can there be a pointer here back to the platform spec? I **think** that would address @samj1912's comment here: https://github.com/buildpacks/spec/pull/305#pullrequestreview-985204225

Reply:
Something like this? b9e6e34